galaxy/Dockerfile

```diff
@@ -192,6 +192,10 @@ RUN ./scripts/common_startup.sh && \
 # Switch back to User root
 USER root
 
+# Copy error pages and make the permissions right for NGINX to access them
+ADD ./500.html /root/500.html
+RUN chmod o+x / && chmod o+x /root && chmod o+x /root/500.html
```

Comment:
This is probably not a good idea. Consider storing the page in a location where it fits better.